committees/models.py

```diff
@@ -0,0 +1,79 @@
+import datetime
+
+from django.db import models
+from django.contrib.auth.models import Group
+
+from core.fields import MarkdownTextField
+from membership_file.models import Member
+
+
+class AssociationGroup(models.Model):
+    site_group = models.OneToOneField(Group, on_delete=models.CASCADE)
+    shorthand = models.CharField(max_length=16, blank=True, null=True)
+    icon = models.ImageField(upload_to='images/committees/', blank=True, null=True)
+
+    COMMITTEE = 'C'
+    GUILD = 'O'
+    WORKGROUP = 'WG'
+    BOARD = 'B'
+    GROUPTYPES = [
+        (COMMITTEE, 'Committee'),
+        (GUILD, 'Guild'),
+        (WORKGROUP, 'Workgroup'),
+        (BOARD, 'Board'),
+    ]
+    type = models.CharField(max_length=2, choices=GROUPTYPES)
+    is_public = models.BooleanField(default=True)
+
+    short_description = models.CharField(max_length=128, blank=True, null=True)
+    long_description = models.TextField(blank=True, null=True)
+
+    # Internal data
+    members = models.ManyToManyField(
+        Member,
+        through='AssociationGroupMembership',
+    )
+
+    contact_email = models.EmailField(null=True, blank=True)
+    instructions = MarkdownTextField(blank=True, null=True, max_length=2047,
+                                    help_text="Information displayed on internal info page")
+
+    class Meta:
+        ordering = ("shorthand",)
+
+    @property
+    def name(self):
+        return self.site_group.name
+
+    def __str__(self):
+        return self.site_group.name
+
+
+class GroupExternalUrl(models.Model):
```

Comment:
I doubt that those links will need to be visited very often (if they do, people will bookmark them instead of accessing them through Squire). Stripping them out completely is more work though, so I suggest to leave them in anyways :P